### PR TITLE
Foreman 2.3 updates

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: vagrant/config/versions.yaml [hunk 1]
```diff
@@ -87,6 +87,18 @@ installers:
       - 'debian10'
       - 'ubuntu1804'
 
+  - foreman: '2.3'
+    katello: 'nightly'
```
Comment (ekohl) [inline vagrant/config/versions.yaml:91]:
```suggestion
    katello: '3.18'
```

File: vagrant/config/versions.yaml [hunk 2]
```diff
@@ -87,6 +87,18 @@ installers:
...
+    pulp: '2.21'
+    pulpcore: '3.6'
```
Comment (ekohl) [inline vagrant/config/versions.yaml:93]: Not sure, but I think they want 3.7. We can fix that later since nightly is also still on 3.6.

**Review by ekohl — CHANGES_REQUESTED** [review]
Note that our nightly upgrade tests will break after this is merged, until RC1 is available and installable.

File: vagrant/config/versions.yaml (hunk 2 above)
Comment (pcreech) [reply vagrant/config/versions.yaml:93]: +1 to fix it after nightly.

File: vagrant/config/versions.yaml (hunk 1 above)
Comment (pcreech) [reply vagrant/config/versions.yaml]: should be fixed

**Review by ekohl — APPROVED** [review]
Approving but not merging so it can be done in sync with other tasks. Feel free to merge when you think is the right time.

File: vagrant/config/versions.yaml (hunk 2 above)
Comment (ekohl) [reply vagrant/config/versions.yaml:93]: @jlsherrill FYI

**Comment by tbrisker** [comment]
RC1 is close now, merging

**Pull request merged**